packages/components/src/DateTimePickers/pickers/MonthPicker/MonthPicker.component.js

```diff
@@ -1,45 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { chunk } from 'lodash';
+import addMonths from 'date-fns/add_months';
+import format from 'date-fns/format';
 import theme from './MonthPicker.scss';
 import PickerAction from '../../PickerAction';
 
+const baseDate = new Date(0);
+const indexes = (new Array(12))
+	.fill(0)
+	.map((_, i) => i);
+
+const months = indexes.map(index => ({
+	index,
+	name: format(addMonths(baseDate, index), 'MMMM'),
+}));
+const monthsRows = chunk(months, 3);
+
 function MonthPicker(props) {
-	const months = [
-		'January',
-		'February',
-		'March',
-		'April',
-		'May',
-		'June',
-		'July',
-		'August',
-		'September',
-		'October',
-		'November',
-		'December',
-	];
-
-	const monthsRows = chunk(months, 3);
-	const selectedMonth = 'September';
-
-	function isSelected(m) {
-		return selectedMonth === m;
-	}
+	const isSelected = index =>
```

Comment:
Please use `function`. Let's keep our codebase consistent

Reply:
or let move to a class so this becomes a method

Reply:
✔